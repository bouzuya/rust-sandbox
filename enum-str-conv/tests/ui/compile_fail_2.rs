fn main() {
    #[derive(enum_str_conv::EnumStrConv)]
    // No error type or unknown fn is provided
    enum MyEnum {
        #[enum_str_conv(str = "apple")]
        A,
    }
}
