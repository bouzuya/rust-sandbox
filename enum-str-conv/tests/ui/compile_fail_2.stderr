error: expected attribute: #[enum_str_conv(error = ..., unknown = ...)]
 --> tests/ui/compile_fail_2.rs:4:5
  |
4 | /     enum MyEnum {
5 | |         #[enum_str_conv(str = "apple")]
6 | |         A,
7 | |     }
  | |_____^
